src/js/components/Calendar/doc.js

```diff
@@ -20,6 +20,12 @@ export const doc = Calendar => {
 
   DocumentedCalendar.propTypes = {
     ...genericProps,
+    activeDate: PropTypes.oneOf(['start', 'end'])
+      .description(
+        `Whether the next date selection will affect the start or end bound of 
+        the range.`,
```

Comment:
Can we add a clarification that this prop is only relevant for `range` users? I know we imply it at the end of the line, but I'm thinking we should be more clear about that right at the beginning. Otherwise the context of 'next date selection' doesn't make sense if you are not using `range`.